resources/scss/app.scss

```diff
@@ -636,3 +636,126 @@ svg > a:hover {
       content: '\f031';
     }
 }
+/**
+* Equipment Card Component
```

Comment:
Can we move these into a partial sass file.

Reply:
Done